src/fielddef.ts

```diff
@@ -156,6 +156,8 @@ export function field(fieldDef: FieldDef, opt: FieldRefOption = {}) {
 
   if (opt.datum) {
     field = `datum["${field}"]`;
+  } else if (field !== undefined && field.indexOf('.') !== -1) { // dot in fieldname
```

Comment:
> I think asking people to escape with '' is completely reasonable.


Im confused.   So are you saying that we do not need to change anything here, but rather update the document to clarify this?